docs/aero_force_inversion_guide_zh.md

```diff
@@ -0,0 +1,126 @@
+# 飞行数据反演气动力（可落地流程）
```

Comment:
We wouldn't take this. 

IF this is supposed to be in the documentation it must be written first in english under the docs/en/ tree (depending on what this is) and then would be translated to Chinese within Crowdin.